modules/library/main/src/main/java/org/geotools/styling/DefaultResourceLocator.java

```diff
@@ -84,9 +82,14 @@ protected URL validateRelativeURL(URL relativeUrl) {
         return null;
     }
 
-    URL makeRelativeURL(String uri) {
+    URL makeRelativeURL(String uri, String query) {
```

Comment:
You are correct, will need to fix that method next. If that implementation used an @overrides annotation it would be broken with a compile error about now ...